runtime/lua/vim/treesitter.lua

```diff
@@ -44,50 +46,28 @@ function Parser:set_included_ranges(ranges)
   self.valid = false
 end
 
-local M = {
-  parse_query = vim._ts_parse_query,
-}
+-- TODO(vigoux): not that great way to do it, but that __index method bothers me...
+local M = vim.tbl_extend("error", query, lang)
```

Comment:
I'm not sure I understand the TODO / problem here.

Reply:
In the end I fixed it, will remove it